### PR TITLE
Fix custom-loader for recent versions of node

### PR DESCRIPTION
**Pull request by dstockwell**

### CODE REVIEW

File: tools/custom-loader.mjs [hunk 1]
```diff
@@ -18,7 +18,7 @@ export function resolve(specifier, parent, resolve) {
   if (specifier.includes('-web.js')) {
     specifier = specifier.replace('-web.js', '-node.js');
   }
-  const resolved = new url.URL(specifier, parent);
+  const resolved = new url.URL(specifier, parent || 'file:///');
```
Comment (shaper) [inline tools/custom-loader.mjs]: When I was doing the Mac hackery with @yuinchien I had to add '/localhost' at one point to progress, but, I was using 'file://' (two slashes, not three), so 'file://localhost/', so I will vote for your solution here (figuring also that you likely actually tested it and progressed farther than I did), but wanted to mention this in any case.

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,4 @@
 language: node_js
-node_js:
-  - "9.5"
```
Comment (shaper) [inline .travis.yml]: What will Travis do now node-js-version-wise? Use the latest stable?

Comment (dstockwell) [reply .travis.yml]: That's what I thought. I had added this as a workaround until we fixed the issue, but now we seem to be getting node "0.10". So I'll set this to just "9" to get the latest 9.* hopefully.

File: tools/custom-loader.mjs (hunk 1 above)
Comment (shaper) [reply tools/custom-loader.mjs]: Also I was fiddling with the specifier, not the parent...

Comment (dstockwell) [reply tools/custom-loader.mjs]: Right, I think `file://localhost/path` and `file:///path` are almost equivalent, the host name should not matter.

**Pull request merged**